.github/workflows/release.yml

```diff
@@ -26,16 +34,61 @@ jobs:
           node-version: 'v18.20.2'
           registry-url: 'https://registry.npmjs.org'
 
+      # Setup a Docker "buildx" builder container, used by "build-push-action"
+      # below for multi-platform image builds. Notes on multi-platform images:
+      # https://github.com/elastic/apm-agent-nodejs/issues/4038#issuecomment-2130406402
+      - uses: docker/setup-buildx-action@v3
+
+      - name: Log in to the Elastic Container registry
+        uses: docker/login-action@9780b0c442fbb1117ed29e0efdff1e18412f7567 # v3.3.0
+        with:
+          registry: ${{ secrets.ELASTIC_DOCKER_REGISTRY }}
+          username: ${{ secrets.ELASTIC_DOCKER_USERNAME }}
+          password: ${{ secrets.ELASTIC_DOCKER_PASSWORD }}
```

Comment:
@elastic/observablt-robots Hi. I'm adding Docker image publishing to this repo for the first time (based on [similar code from apm-agent-node.js git](https://github.com/elastic/apm-agent-nodejs/blob/main/.github/workflows/release.yml). Could you please help me get these secrets added to this repo, if they are not already there? Thanks.

Reply:
They are now available